index.js

```diff
@@ -92,11 +94,21 @@ if (config.language !== 'node_js') {
 
 getVersions(config)
 	.map(function (version) {
+		state[version] = states.preparing;
+		updateState(state);
+
+		return version;
+	})
+	.tap(function () {
+		return Promise.resolve(config.services).map(service.pull);
+	})
+	.map(function (version) {
 		var context = {
-			version: version,
+			version: format('%s-onbuild', version),
```

Comment:
Because I wanted to make `pullImage` reusable for my services. The `-onbuild` was hardcoded there.

I guess we should refactor that part to make it somewhat like the services. Create a map of supported languages with settings (like version, repository, ...). That will remove the `-onbuild` here.